data/make_data.py

```
@@ -0,0 +1,160 @@
+TODO = [0, data_len - 1]
+PREVIEW_LIMIT = 100
+for idx in TODO:
+    ptr, size = data._index[idx]
+    dix = data.get(idx=idx, offset=0, length=size).astype(int)
+    print("-" * 70 + f"[{OUT_NAME} idx {idx} sz {size}]")
+    assert dix[-1] == 0
+    dix = dix[:-1]
+    if len(dix) > PREVIEW_LIMIT:
+        try:
+            print(tokenizer.decode(dix[:PREVIEW_LIMIT]))
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)